ooxml/XSSF/UserModel/XSSFWorkbook.cs

```diff
@@ -230,8 +230,8 @@ public XSSFWorkbook(OPCPackage pkg)
          *       pkg.close(); // gracefully closes the underlying zip file
          *   </code></pre>     
          */
-        public XSSFWorkbook(Stream is1)
-            : base(PackageHelper.Open(is1))
+        public XSSFWorkbook(Stream fileStream, bool bReadonly=false)
+            : base(PackageHelper.Open(fileStream, bReadonly))
```

Comment:
Such changes can introduce a breaking change. It will be better to mark old one with `Obsolete` attribute and create another one with new arguments. If you're fine with it, it should be included in release notes